frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -14,12 +14,13 @@ export interface MultiColumnFieldRowProps {
   disableDeleteRow?: boolean;
 }
 
-const minusCircleIcon = (onDelete: () => void, disableDeleteRow?: boolean, toolTip?: string) => {
+const minusCircleIcon = (onDelete: () => void, disableDeleteRow?: boolean) => {
   return (
-    <div className={cx('odc-multi-column-field__col--button', { 'is-disabled': disableDeleteRow })}>
-      <MinusCircleIcon aria-hidden="true" onClick={onDelete} />
-      <span className="sr-only">{toolTip || 'Delete'}</span>
-    </div>
+    <MinusCircleIcon
+      className={cx('odc-multi-column-field__col--button', { 'is-disabled': disableDeleteRow })}
+      aria-hidden="true"
+      onClick={onDelete}
+    />
```

Comment:
I think you're better off just leaving this as is. I tested with vox chrome and it read the sr-only text. Didn't try jaws though.
Neither approach is fully correct though because neither is semantically a button or has the appropriate roles.
Doesn't even look like the disabled prop will prevent the onclick :P
I'd just file a new bug if i were you to address any concerns around this button.